src/Components/SearchBox/SearchBox.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+// import styles from './HeroTitle.module.scss';
+import './SearchBox.scss';
+import magnifying_glass from '../../Assets/svg/magnifying_glass.svg'
+
+
+const SearchBox = () => {
+    return(
+        // <div className={styles['hero']}>
+        <div className='search_box_container'>
+
+            <div className='content_container'>
+                <a className='big_font'>Epicure works with the top chef restaurants in Tel Aviv</a>
```

Comment:
- For semantic correctness, it's advisable to use appropriate HTML tags for different types of content. Instead of using {a} tags for titles, consider using tags like h1, h2, etc., for heading elements. This helps improve accessibility and search engine optimization.

```suggestion
    <h2 className='big_font'>Epicure works with the top chef restaurants in Tel Aviv</h2>
```